File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/BudgetReportItem.java
package com.kendoui.spring.models;

public class BudgetReportItem {
    private String unit;
    private int budget;
    private int spending;
    
    public BudgetReportItem(String unit, int budget, int spending) {
        setUnit(unit);
        setBudget(budget);
        setSpending(spending);
    }
    
    public String getUnit() {
        return unit;
    }

    public void setUnit(String unit) {
        this.unit = unit;
    }
    
    public int getBudget() {
        return budget;
    }

    public void setBudget(int budget) {
        this.budget = budget;
    }
    
    public int getSpending() {
        return spending;
    }

    public void setSpending(int spending) {
        this.spending = spending;
    }
}

File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/BudgetReportItem.cs
namespace Kendo.Mvc.Examples.Models
{
    public class BudgetReportItem
    {
        public string Unit { get; set; }
        public int Budget { get; set; }
        public int Spending { get; set; }

        public BudgetReportItem(string unit, int budget, int spending)
        {
            Unit = unit;
            Budget = budget;
            Spending = spending;
        }
    }
}